### DISCUSSION TITLE
Connect Nemo to a digital human Avatar as I do with ChatGPT

### DISCUSSION THREAD

**Discussion by DocJEspo** [NVIDIA-NeMo/NeMo#7422] [Q&A]
Folks, alot of good work done here, trying to find specifics on connecting a Nemo instance to my DH. For example, when I connect to chatGPT I need to pass an API key along with Api type - so if I'm using Azure, I need to pass in resource and deployment name along with api version.... can someone please point me in the right direction for that for Nemo, please? And thank you!

**Comment by InfoVision-US** [comment]
Discover how [digital humans](https://www.infovision.com/blog/fascinating-world-digital-humans-creation-real-world-uses) redefine engagement while preserving the human touch. Explore their impact here: [https://www.infovision.com/blog/fascinating-world-digital-humans-creation-real-world-uses](https://www.infovision.com/blog/fascinating-world-digital-humans-creation-real-world-uses)

#DigitalHumans #VirtualHumans #EmergingTech #HumanComputerInteraction
